packages/db/src/db.ts

```diff
@@ -11,3 +11,10 @@ export type Visit = typeof visits.$inferSelect
 export type JourneyStep = typeof journeySteps.$inferSelect
 export type Museum = typeof museums.$inferSelect
 export type Car = typeof cars.$inferSelect
+
+export type VisitWithJourneyAndMuseum = Visit & {
```

Comment:
This type is unnecessary. It should be located in the mobile directory because it is only used on mobile-side. It doesn't need to be shared. It's just pollution. Please move this in mobile folder.